rwengine/src/engine/GameData.cpp

```diff
@@ -446,6 +446,7 @@ void GameData::loadModel(ModelID model) {
             slotname = name;
             break;
         case ModelDataType::PedInfo:
+        {
```

Comment:
Moved the brace.
I haven't checked but maybe it 'worked' because the variable is static?